include/ck/utility/generic_memory_space_atomic_max.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include "data_type.hpp"
+
+namespace ck {
+
+template <typename X>
```

Comment:
Please see above comment "template declaration (but without definition)"

Reply:
Fix in 220985c9c54594f0d9ae4f73ea2dc5f8eb56e9b5